### ISSUE TITLE
Dashboard: All Views; A11y, Missing H1 

### ISSUE THREAD

**Issue by carloskelly13** [GoogleForCreators/web-stories-wp#4626]
## Feature Description

We replaced the h1 text with an SVG, we need to add in a level-one header, it doesn't need to be visible on the screen but it needs to be present. Alternatively, we can wrap the svg that has the header text as an h1 and give it an aria label and make sure the svg is aria-hidden to avoid double SR readings.

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by swissspidy** [comment]
> Alternatively, we can wrap the svg that has the header text as an h1 and give it an aria label and make sure the svg is aria-hidden to avoid double SR readings.

I _think_ you can just wrap the svg in an h1, and make sure the svg has a `<title>` (by passing a `title` prop). No need for aria labels and the like
